examples/fluent-bit/cloudwatchlogs/README.md

```diff
@@ -31,3 +31,17 @@ This field instructs the CloudWatch plugin to only send the value of the log key
 		}
 	},
 ```
+
+			"log_key": "log",
+The following table shows how your logs will appear in CloudWatch with and without `log_key` and `enable-ecs-log-metadata` if "`my_sample log`" is printed to stdout:
+
+|Add logConfiguration.options|Add firelensConfiguration.options|received by cloudwatch|description|comments|
+|-|-|-|-|-|
+|||{<br>    "container_id": "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX-XXXXXXXXXX",<br>    "container_name": "my_container_name",<br>    "ecs_cluster": "my_cluster_name",<br>    "ecs_task_arn": "arn:aws:ecs:region:9876543210:task/my_task_arn",<br>    "ecs_task_definition": "my_task_definition:revision_number",<br>    "log": "my_sample log",<br>    "source": "stdout"<br>}|no log key is set and ecs-log-metadata is enabled by default||
+||"enable-ecs-log-metadata":"false"|{<br>    "container_id": "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX-XXXXXXXXXX",<br>    "container_name": "my_container_name",<br>    "log": "my_sample log",<br>    "source": "stdout"<br>}|no log key is set and ecs-log-metadata is disabled||
+|"log_key":"log"||"my_sample log"|log key set to "log" and ecs-log-metadata is enabled by default|less efficient|
+|"log_key":"log"|"enable-ecs-log-metadata":"false"|"my_sample log"|log key set to "log" and ecs-log-metadata is disabled|<b>more efficient</b>|
+
+#### CloudWatch configuration options
```

Comment:
Can we put this first before the "What if I just want the raw log line from the container to appear in CloudWatch?" header? I think its the most important thing for users to see first... I don't want them to miss it with the table which is huge...